### DISCUSSION TITLE
Rotating Map libre vector layer

### DISCUSSION THREAD

**Discussion by cubecleveland** [maplibre/maplibre-gl-js#957] [General]
Hi, 
is there any way to rotate the layer (base layer) ?

We are trying to achive a rotation so the user can always see his building upright event thought the layer is tilted.... (for editing purposes)

Any help will be appreciated ?

**Comment by wipfli** [comment]
Press Ctrl, click on map, move mouse should result in rotation and pitch. https://maplibre.org/maplibre-gl-js-docs/example/navigation/

**Comment by cubecleveland** [comment]
is there any way to use it with the maplibre gl leafelt plugin... i have the layer load on a leaflet map
